app/views/ChargeSummary.scala.html

```diff
@@ -381,18 +357,6 @@ <h1 class="govuk-heading-xl">
         }
     }
 
-    @if(viewModel.remainingToPayWithNoCodingOutEnabledAndIsPayeSelfAssessment){
```

Comment:
:recycle: - Please drop this properly from the viewModel as this is not required anymore;